[wxUniv] Don't cross out a disabled wxStaticText or wxButton

The "mono" theme shows the disabled state of a wxButton by crossing it out, but
unfortunately the same happens to wxStaticText which looks really ugly, so
disable the cross for now.

Signed-off-by: Anders Larsen <al@alarsen.net>
---

 src/univ/themes/mono.cpp |    9 ---------
 1 files changed, 0 insertions(+), 9 deletions(-)


diff --git a/src/univ/themes/mono.cpp b/src/univ/themes/mono.cpp
index 93cf4b9..f4b01bf 100644
--- a/src/univ/themes/mono.cpp
+++ b/src/univ/themes/mono.cpp
@@ -730,15 +730,6 @@ void wxMonoRenderer::DrawButtonLabel(wxDC& dc,
                                      wxRect *rectBounds)
 {
     dc.DrawLabel(label, image, rect, alignment, indexAccel, rectBounds);
-
-    if ( flags & wxCONTROL_DISABLED )
-    {
-        // this is ugly but I don't know how to show disabled button visually
-        // in monochrome theme otherwise, so cross it out
-        dc.SetPen(m_penFg);
-        dc.DrawLine(rect.GetTopLeft(), rect.GetBottomRight());
-        dc.DrawLine(rect.GetTopRight(), rect.GetBottomLeft());
-    }
 }
 
 // ----------------------------------------------------------------------------
